src/commands/server/update.ts

```diff
@@ -151,4 +156,12 @@ export default class Update extends Command {
 
     this.exit(0)
   }
+
+  async setDomainFlag(flags: any): Promise<void> {
+    const kubeHelper = new KubeHelper()
+    const cheCluster = await kubeHelper.getCheCluster('eclipse-che', flags.chenamespace)
```

Comment:
`eclipse-che` must be extracted into constant to make it possible easier customize in crwctl

Reply:
It does not seem to be a universal way to extract domain for all installer, it would be better to make it a part of `installers/operator.ts`

Reply:
It is related to platform, not to installer type

Reply:
Really?
I believe it's related to Operator installer
Helm chart does not have such Custom Resource
But I may be wrong

Reply:
It is used to deploy Eclipse Che
https://github.com/che-incubator/chectl/blob/ab/domainupdate/src/tasks/installers/helm.ts#L331

Reply:
Ok, you mean domain flag - and I agree with that, but actually I mean using CustomResource Che Cluster...
The way how the domain is evaluated on already deployed Che - depends on the installer.
For Operator - it's CustomResource CheCluster
For Helm Chart - it's directly Che Config Map, right?

Reply:
Right, but we don't support update in case of helm installer.